src/System.Runtime/tests/Performance/Perf.String.cs

```diff
@@ -3,102 +3,133 @@
 
 using System.Text;
 using System.Collections.Generic;
-using Xunit;
+
 using Microsoft.Xunit.Performance;
+using Xunit;
 
 namespace System.Runtime.Tests
 {
-    public class Perf_String
+    public static class Perf_String
     {
-        public static IEnumerable<object[]> TestStringSizes()
+        public static IEnumerable<object[]> StringSizesTestData()
         {
             yield return new object[] { 10 };
             yield return new object[] { 100 };
             yield return new object[] { 1000 };
         }
 
-        [Benchmark]
-        [MemberData("TestStringSizes")]
-        public void GetChars(int size)
+        [Benchmark , MemberData("StringSizesTestData")]
```

Comment:
I just follow what the majority of tests in the project follow - in this case its the new one liner